### ISSUE TITLE
Bower packages: query do not return versions

### ISSUE THREAD

**Issue by raquelsa** [librariesio/support#85]
Hi,

Any plans on adding "versions" info for the bower packages queries?

Thanks!

**Comment by andrew** [comment]
@raquelsa yeah eventually, need to transition the data from git tags to versions which may take a little while.
